front/lib/api/assistant/conversation.ts

```diff
@@ -199,6 +207,10 @@ export async function deleteConversation(
     throw new Error(`Conversation ${conversationId} not found`);
   }
 
+  if (!canAccessConversation(auth, conversation)) {
```

Comment:
preserving prior behaviour here, this is why i don't pass a Result as in other functions

Reply:
Not relevant anymore, I guess?